docs/CONTRIBUTING.md

```diff
@@ -75,10 +67,27 @@ Below is a list of other guidelines we try to follow for PRs.
   - As an exception to this rule, a reviewer who has just approved a PR can decide to rebase the branch via the GitHub UI if
     - the author has enabled auto-merge and
     - a branch update (without conflicts) is the only thing stopping the auto-merge
-- For substantial changes, we prefer creating a "parent/feature" PR which receives multiple smaller PRs. For the parent PR and all the smaller child PRs, all the general guidelines apply.
-- For PRs towards main
-  - Every PR should contain a reference to a devops task
+- PRs should have "the shortest life possible" and therefore be merged to `main` as quickly as possible, to avoid the overhead of maintenance. As a consequence:
+  - We make PRs as small as possible
+  - We prioritize reviewing code over writing new code
+  - We try to split larger changes/PRs into smaller chunks/refactors that can be merged independently
+  - We try to avoid mixing frontend & backend changes in one PR, whenever possible.
+    - For example, if the backend adds a new endpoint, then the frontend code that uses that new endpoint should be added in a separate PR towards `main`, that is reviewed once the endpoint itself has already been merged to `main`. This is opposed to the approach of creating a feature branch that contains both the backend and the frontend code for this new feature.
+    - As an example of when this is unavoidable: when the backend changes an existing endpoint in a way that breaks the frontend, then the adjustment in the frontend needs to be included in the same PR.
+- We avoid "feature branches". PRs towards `main` that are a result of different authors are still possible, but
```

Comment:
We don't define what a "feature branch" (including quotes) is so not doing them becomes more vague. I'm guessing you mean "multi-author PRs towards `main`"?

Reply:
I moved the feature branch stuff into it's own section to hopefully clarify